### ISSUE TITLE
`surround`: backward selection gives range error

### ISSUE THREAD

**Issue by acidghost** [esm7/obsidian-vimrc-support#91]
When doing a backward selection (i.e. `v`, then `b` a couple of times and then `[[`) I get the following error:

<img width="1149" alt="Screenshot 2022-04-05 at 14 40 31" src="https://user-images.githubusercontent.com/1787979/161755784-40d46f9c-0000-4b11-aa04-241ed0c079e3.png">

Here's my vimrc:
```vim
" Have j and k navigate visual lines rather than logical ones
nmap j gj
nmap k gk
" Quickly remove search highlights
nmap @ :nohl

" Yank to system clipboard
set clipboard=unnamed

" Go back and forward with Ctrl+O and Ctrl+I
" (make sure to remove default Obsidian shortcuts for these to work)
exmap back obcommand app:go-back
nmap <C-o> :back
exmap forward obcommand app:go-forward
nmap <C-i> :forward

" Surround text
exmap wiki surround [[ ]]
map [[ :wiki
exmap paren surround ( )
map [( :paren
```

Obsidian version: v0.14.2
Plugin version: 0.6.0

**Comment by esm7** [comment]
I did not take backward selection into account when writing the part of this functionality that was incompatible with CM6. Will fix.

**Comment by esm7** [comment]
Should be fixed in 0.6.1.

**Comment by acidghost** [comment]
Can confirm it is, thanks!
